src/formio/components/email.ts

```diff
@@ -11,6 +11,7 @@ export type EmailInputSchema = InputComponentSchema<string, Validator, Translata
  */
 export interface BaseEmailComponentSchema extends Omit<EmailInputSchema, 'hideLabel' | 'disabled'> {
   type: 'email';
+  validateOn?: 'blur';
```

Comment:
Since it's always set, I would just make this a fixed value (see https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/form/email.js#L19)

```suggestion
  validateOn: 'blur';
```

that will require updating the tests too though! (including the `expectNotAssignable`)